esp_encrypted_img/idf_component.yml

```diff
@@ -0,0 +1,6 @@
+dependencies:
```

Comment:
Please reorder this file per https://github.com/espressif/idf-extra-components/blob/master/sh2lib/idf_component.yml for consistency